cmake/external/abseil-cpp.cmake

```diff
@@ -14,21 +11,13 @@ FetchContent_Declare(
     abseil_cpp
     PREFIX "${CMAKE_CURRENT_BINARY_DIR}/abseil-cpp"
     BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/external/abseil-cpp"
-    GIT_REPOSITORY ${abseil_URL}
-    GIT_TAG ${abseil_TAG}
```

Comment:
If we use a GIT_TAG, then PATCH_COMMAND will run every time when you execute "cmake", even when the source is already patched.

Reply:
Do you know what's the cmake version? This may have been fixed: https://gitlab.kitware.com/cmake/cmake/-/issues/21086